meetings/2023-02-01.md

```diff
@@ -50,36 +57,125 @@ URL: https://blog.gitter.im/2023/01/16/gitter-is-going-fully-native-matrix-in-fe
+* SC: Are we good to go ahead and see what happens? 
+* TBL: I think the answer is yes.
+* SC: Then let's resolve. 
+* eP: (from jitsi chat)
+  * To be clear: Gitter will not be going away. All of your Gitter rooms and content will be accessible via Matrix and the gitter.im
+  * homeserver will continue to live on and be open for everyone to sign-in, interact, and chat like normal. Gitter has effectively been uploaded into (the) Matrix! 
+  * "Because we will be invalidating all access tokens on Gitter, this means that any bots/scripts that use the Gitter API will stop working. You can prepare for the new Matrix world by adapting your code to use https://spec.matrix.org/latest/client-server-api/
+  * You won’t have access to all of your Gitter data to test against before the cut-over but testing against your own Matrix sandbox room should be sufficient for any use case." 
+ 
 
 ### Solid-based chat for CG Chat
 * SC: We can work on a plan to use a Solid-based chat for the CG. Some discussion:
   * https://lists.w3.org/Archives/Public/public-solid/2023Jan/0015.html and thread.
   * https://gitter.im/solid/chat?at=63d95207fb5fe8552e594d49 and after.
 * SC: Thoughts?
-
+* SC: Solid OS Team Chat: https://solidos.solidcommunity.net/Team/SolidOs%20team%20chat/
+* SC: There are questions about features and how suitable it is, how much support it will have, who owns the storage. 
+* eP: I do support having further down the roadmap switching to Solid. I have two requirements: official storage is solidproject.org, and that there are at least 3 clients and at least one of them works on mobile. 
+* TBL: If it was on solidproject.solidcommunity.net, would that work?
+* eP: We can discuss but I'd like solidproject.org. 
+* SC: I think what's important is the commitment to ??? 
+* TBL: solidcommunity.net and solidproject.org are both maintained by the Solid team.
+* SC: We discussed some of that on Gitter yesterday, but I want to acknowledge eP's suggestions and requirements. But all that we should have written down and revisit. 
+* TBL: is it okay for mobile app to be a web app?
+* eP: a Progressive Web App works fine. 
+* SC: Can I ask for this to be passed on to the Solid Chat team? Can the team document what's possible and what they're willing to support with?
+* TBL: We have one in SolidOS, Jackson made one.. 
+* SC: Would you put this on SolidOS team agenda? Or should I join a meeting and suggest? 
+* TBL: Maybe make an issue in SolidOS to create a proposal. 
+* SC: Ok. 
+
+ACTION: SC to create an issue on SolidOS to create a proposal for Solid Chat. 
+
+* WT: I wanted to confirm that I think this is not a priority for CG/WG. If we have a chat that works, it works, and if it's to be a showcase of Solid, which I would support, then I think it can't just be one central chat client over one server storing all the data. we should have multiple clients by multiple developers with data storage in user's pods for it to be showcased at all. 
```

Comment:
```suggestion
* WT: I wanted to confirm that I think this is not a priority for CG/WG. If we have a chat that works, it works, and if it's to be a showcase of Solid, which I would support, then I think it can't just be one central chat client over one server storing all the data. we should have multiple clients by multiple developers with data storage in users' pods for it to be showcased at all. 
```